### PR TITLE
Pioneer 18c calibration csvs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09029__20220417.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9029,CC_calibration_date,1650153600000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09029__20220417.csv:2]: good

File: calibration/CTDGVM/CGINS-CTDGVM-09307__20210721.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9307,CC_calibration_date,1626825600000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09307__20210721.csv:2]: good

File: calibration/CTDGVM/CGINS-CTDGVM-09347__20220508.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9347,CC_calibration_date,1651968000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09347__20220508.csv:2]: good

File: calibration/FLORTM/CGINS-FLORTM-03921__20220427.csv [hunk 4]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3921,CC_angular_resolution,1.076,
+3921,CC_depolarization_ratio,0.039,
+3921,CC_measurement_wavelength,700.0,
+3921,CC_scattering_angle,124.0,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-03921__20220427.csv:5]: good

File: calibration/FLORTM/CGINS-FLORTM-04093__20210715.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4093,CC_angular_resolution,1.076,
+4093,CC_depolarization_ratio,0.039,
+4093,CC_measurement_wavelength,700,
+4093,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04093__20210715.csv:5]: good

File: calibration/FLORTM/CGINS-FLORTM-04201__20220427.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4201,CC_angular_resolution,1.076,
+4201,CC_depolarization_ratio,0.039,
+4201,CC_measurement_wavelength,700,
+4201,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04201__20220427.csv:5]: good

File: calibration/FLORTO/CGINS-FLORTO-01348__20210715.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1348,CC_angular_resolution,1.076,
+1348,CC_depolarization_ratio,0.039,
+1348,CC_measurement_wavelength,700,
+1348,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTO/CGINS-FLORTO-01348__20210715.csv:5]: good

File: calibration/PARADM/CGINS-PARADM-50145__20220502.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50145,CC_bsipar_par_scaling,1,constant
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50145__20220502.csv:2]: good

File: calibration/PARADM/CGINS-PARADM-50218__20211004.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50218,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50218__20211004.csv:2]: good

File: calibration/PARADM/CGINS-PARADM-50304__20220502.csv [hunk 10]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50304,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50304__20220502.csv:2]: good

**Review by rtravis-whoi — APPROVED** [review]

**Review by rtravis-whoi — APPROVED** [review]
the 3 added DOSTA files look good but for some reason I can't add individual review comments to each.

File: calibration/DOSTAM/CGINS-DOSTAM-00364__20220309.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+364,CC_calibration_date,1646784000000,sensor calibration date (milliseconds since 1970-01-01)
+364,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00364__20220309.csv:3]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00195__20220309.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+195,CC_calibration_date,1646784000000,sensor calibration date (milliseconds since 1970-01-01)
+195,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00195__20220309.csv:3]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00170__20220309.csv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+170,CC_calibration_date,1646784000000,sensor calibration date (milliseconds since 1970-01-01)
+170,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00170__20220309.csv:3]: good

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-09029__20220417.csv (hunk 1 above)
Comment (reedan88) [reply calibration/CTDGVM/CGINS-CTDGVM-09029__20220417.csv:2]: good

File: calibration/CTDGVM/CGINS-CTDGVM-09347__20220508.csv (hunk 3 above)
Comment (reedan88) [inline calibration/CTDGVM/CGINS-CTDGVM-09347__20220508.csv:2]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00170__20220309.csv (hunk 13 above)
Comment (reedan88) [inline calibration/DOSTAM/CGINS-DOSTAM-00170__20220309.csv:3]: good

File: calibration/FLORTM/CGINS-FLORTM-03921__20220427.csv (hunk 4 above)
Comment (reedan88) [inline calibration/FLORTM/CGINS-FLORTM-03921__20220427.csv:5]: good

File: calibration/PARADM/CGINS-PARADM-50145__20220502.csv (hunk 8 above)
Comment (reedan88) [inline calibration/PARADM/CGINS-PARADM-50145__20220502.csv:2]: good

File: calibration/PARADM/CGINS-PARADM-50304__20220502.csv (hunk 10 above)
Comment (reedan88) [inline calibration/PARADM/CGINS-PARADM-50304__20220502.csv:2]: good

**Pull request merged**